packages/contentful--create-contentful-app/lib/index.js

```diff
@@ -94,7 +94,7 @@ async function initProject(appName, options) {
   program
     .command('init', { isDefault: true, hidden: true })
     .description('Bootstrap your app inside a new folder ‘app-name’')
-    .argument('[app-name]', 'App name')
+    .argument('[app-name]', 'App name', 'my-app')
```

Comment:
Didn't we say we want to start interactive mode when it's missing?

Reply:
This will be done in the follow-up PR